pipeline-modules/common/aws/git-clone.yaml

```
@@ -36,3 +41,13 @@ template: |
     build:
       commands:
         - git-clone.sh {{ git_remote_path }} {{ git_checkout_target }} {{ git_local_path }}
+        - cd {{ working_dir }}
+        - if [ -f "terrarium.yaml" ]; then
+           echo "terrarium.yaml exists. Encoding and adding to outputs...";
+           terrarium_yaml_base64=$(base64 -w 0 terrarium.yaml);
+          else
+           echo "terrarium.yaml does not exist. Skipping encoding and adding to outputs.";
+           terrarium_yaml_base64="";
+          fi
+        - var_prefix="terrarium_yaml_base64"
+        - outputs.sh add "$var_prefix" "$terrarium_yaml_base64"
```

only set the output if yaml is present.

```suggestion
        - if [ -f "terrarium.yaml" ]; then
           echo "terrarium.yaml exists. Encoding and adding to outputs...";
           terrarium_yaml_base64=$(base64 -w 0 terrarium.yaml);
           outputs.sh add "terrarium_yaml_base64" "$terrarium_yaml_base64"
          else
           echo "terrarium.yaml does not exist. Skipping encoding and adding to outputs.";
           terrarium_yaml_base64="";
          fi
```

---

according to the discussion, there were some issues when trying to move it inside the if condition and this is working fine.